examples/rerun/lang_tests/rerun_stdout.py

```diff
@@ -0,0 +1,18 @@
+# VM:
+#   status: success
+#   stdout: a
+#   rerun-if-stdout: b
+
+import os, sys
+
+cookie = os.path.join(os.environ["CARGO_MANIFEST_DIR"], "target", "tmp", "rerun_stdout_cookie")
+i = 0
+if os.path.exists(cookie):
+    i = int(open(cookie, "r").read().strip())
```

Comment:
Are we OK with having two file handles to the cookie open concurently? Should we use a context manager to close the first? No idea.

Reply:
> Are we OK with having two file handles to the cookie open concurently? 

We aren't! That's why each test uses a separate cookie file: without that, the three tests would fail in weird ways.

Reply:
Cookie files aren't opened concurrently.

Reply:
```
open(cookie, "r")
```

with no corresponding `.close()`, and then:

```
open(cookie, "w")
```

later.

What am I missing?

Reply:
I see what you mean! Due to the wonders of reference counting, those won't be concurrent. And, even if they were, the write would (correctly) create a new inode which wouldn't interfere with the read. So we're OK either way.

Reply:
I did not know that Python closed files when the ref count drops to zero! Begs the question: why to use `with open() as f: ...`

>  even if they were, the write would (correctly) create a new inode which wouldn't interfere with the read

I don't think that's right, but it's also irrelevant now.